sda/cmd/ingest/ingest.go

```diff
@@ -100,6 +101,24 @@ func main() {
 	var message schema.IngestionTrigger
 
 	go func() {
+		start := time.Now()
+		for i := 0; i <= 10; i++ {
+			h, err := db.ListKeyHashes()
+			if err != nil {
+				log.Errorln(err.Error())
+			}
+			if len(h) != 0 {
+				break
+			}
+
+			time.Sleep(time.Duration(30 * time.Second))
+			if time.Since(start).Seconds() >= float64(300) {
+				log.Errorln("no crypt4gh key hash registered, restarting")
+				forever <- false
+			}
+			log.Errorln("no crypt4gh key hash registered")
```

Comment:
This message will be repeated, that is, for every 30 seconds, the message is printed twice. Move it to the line in front of `forever <- false`

Reply:
It's printed once every 30s for 300s then the container will restart.